### ISSUE TITLE
🚀[FEA]: Finetuning Models

### ISSUE THREAD

**Issue by SaptarishiD** [NVIDIA/earth2mip#202]
### Is this a new feature, an improvement, or a change to existing functionality?

New Feature

### How would you describe the priority of this feature request

Medium

### Please provide a clear description of problem you would like to solve.

Is it possible for earth2mip to have some functionality that allows us to finetune any model to our own data or even customize the loss function of the model so that it achieves better predictions over a particular region?

**Comment by nbren12** [comment]
This is a bit out of scope of the MIP, which focuses on inference, not training.

That said it should be possible to fine tune the pytorch based models packaged in this repo and in https://github.com/nvidia/modulus.
